index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="jquery/jquery-min.js"></script>
+  <script src="bootstrap-4.6.1-dist/js/bootstrap.bundle.min.js"></script>
+  <script type="text/javascript" src="bootstrap-4.6.1-dist/js/bootstrap.min.js"></script>
+  <link rel="stylesheet" href="bootstrap-4.6.1-dist/css/bootstrap.min.css">
+  <link rel="stylesheet" href="boxicons/css/boxicons.min.css">
+  <link rel="stylesheet" href="style.css">
+  <title>Proton..</title>
+</head>
+
+<body>
+  <header>
+    <div class="container-fluid">
+      <nav class="navbar navbar-expand-lg navbar-dark fixed-top">

```

Comment:
- [ ] Please always try to explain the reasoning for your suggestion (what are the advantages of putting the logo outside of the `<nav>` tag !!). it may not be obvious to the reader why you are making this suggestion.

> For example you can say
> Great job using bootstrap classes 💯👌, I think It would be better to put the logo outside of the `<nav>` tag, Using semantic tags correctly make it clear to the browser what the meaning of a page and its content is. That clarity is also communicated with search engines, ensuring that the right pages are delivered for the right queries.

Reply:
than you!

so what do I do next, do I update the review?

Reply:
Yes, Update it, and submit the same link (PR) 👌